Libraries/Components/TextInput/TextInput.js

```diff
@@ -152,6 +152,10 @@ const DataDetectorTypes = [
  * );
  * ```
  *
+ * By default, when you assign a `height` style to TextInput, the text & placeholder
+ * will render at the top for iOS, but the center for Android, to fix this, 
+ * set `textAlignVertical: "top"` in the style prop. See the Multiline example below.  
```

Comment:
I would change double quotes to single quotes which is common in javascript. so like this `textAlignVertical: 'top'`